### DISCUSSION TITLE
Video intro of contextive at fsharpConf 2023

### DISCUSSION THREAD

**Discussion by FH-Inway** [dev-cycles/contextive#68] [Show and tell]
Came to Contextive today via [TDD & DDD from the Ground Up (Live Coding) - Chris Simon - NDC Sydney 2024](https://www.youtube.com/watch?v=IP71G44pQLE) (which is also very nice) and liked the idea and how it is implemented very much.

After some digging, I found [Ubiquitous F# in Contextive, the Ubiquitous Language Tool | fsharpConf 2023](https://www.youtube.com/watch?v=bf1as7nTFEY), which so far is the best intro to the tool. I wanted to highlight it here in case others are looking for a resource to get to know Contextive.

**Comment by chrissimon-au** [comment]
Thanks for sharing! You might also enjoy [Managing Domain Knowledge with Chris Simon](https://www.youtube.com/watch?v=uk7hCIRJmW4).  The Contextive demo at the end is similar to the FSharpConf one, but the first 45mins is a more philosophical 'systems thinking' look at knowledge management and documentation generally.

This is a good prompt, I should probably include these videos in the README and documentation pages.

**Comment by chrissimon-au** [comment]
Thanks for the suggestion - I've now added a short 15min video from NDC Sydney 2024 interview with SSW to the home README and have added a VIDEOS page with a bunch more videos that I've done demonstrating and talking about Contextive: https://github.com/dev-cycles/contextive/blob/main/docs/wiki/VIDEOS.md
